components/slider.js

```diff
@@ -1,23 +1,63 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
-import animateScrollTo from 'animated-scroll-to';
+import scrollToWithAnimation from 'scrollto-with-animation';
 import { Highlight } from '../components/highlights'
 import stylesheet from './slider.scss'
 import { ArrowButton } from './arrow-button';
 import { MediaList } from './icon';
 import { IndustriesMain } from './industries-main';
 
+const autoSliderTime = 15000;
+
 export class Slider extends React.Component {
   constructor(props) {
     super(props);
     this.state = {currentScroll: 0, isBeginning: true, isEnd: false}
     this.scroll = this.scroll.bind(this);
+    this.scrollAutomatically = this.scrollAutomatically.bind(this);
     this.list = this.props.listRef;
+    this.shouldScrollRight = true;
 
     this.setListRef = element => {
       this.list = element;
     };
+
+    if(this.props.shouldScrollAutomatically){
+      this.scrollAutomatically();
+    }
+  }
+
+  scrollAutomatically(){
+    const element = this.list;
+    if(!element){
+      setTimeout(() => this.scrollAutomatically(), 500);
+      return;
+    }
+    const scrollWidth = element.scrollWidth;
+
+    if(this.shouldScrollRight){
+      scrollToWithAnimation(
+        element,
+        'scrollLeft',
+        scrollWidth,
+        autoSliderTime, 
+        'linearTween',      
+      );
+    }
+    else {
+      scrollToWithAnimation(
+        element,
+        'scrollLeft',
+        scrollWidth * -1, 
+        autoSliderTime,
+        'linearTween',
+     );
+    }
+
+    this.shouldScrollRight = !this.shouldScrollRight;
```

Comment:
This toggling of that value makes me wonder whether the name of the variable is the most accurate one.

Reply:
Related to what it translates to in practice. At any given point the slider is either scrolling left or right. That's how the name came up. I can do something like, "scrollingForward"? Perhaps that is more self-explanatory, as its more of a verb and action. Should be easier to make sense of.